### DISCUSSION TITLE
Report widget is not editable, need to hide non-called API instead of displaying them as 0.

### DISCUSSION THREAD

**Discussion by YixuanEric** [Azure/api-management-developer-portal#2593] [Feature request]
In the report wdiget, can we have this report only shows the APIs/products if they actually have been called?
For those if the call number is “0”, don’t show them in the list.  This will simplify the page view and easy to read.

**Currently we cannot edit this widget, is there any ETA of enabling users to edit the report?**

![image](https://user-images.githubusercontent.com/17732736/160968085-66e5dfd4-5c4a-4d1b-91ec-1e124ba7c1f2.png)

**Comment by ghost** [comment]
@YixuanEric, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@YixuanEric, thank you for requesting this feature.

**Comment by azaslonov** [comment]
Unfortunately, we can't provide an ETA at this moment.
